pkg/utils/numbers_test.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
Nit: Since the file has been touched
```suggestion
// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
```